cluster/node-pools/worker-default/stack.yaml

```diff
@@ -74,15 +74,15 @@ Resources:
         Value: ready
       - Key: k8s.io/cluster-autoscaler/node-template/label/aws.amazon.com/spot
         PropagateAtLaunch: true
-        Value: {{if or (eq .NodePool.DiscountStrategy "spot") (eq .NodePool.DiscountStrategy "spot_max_price")}}true{{else}}false{{end}}
+        Value: {{if .NodePool.IsSpot }}true{{else}}false{{end}}
       - Key: k8s.io/cluster-autoscaler/node-template/resources/ephemeral-storage
         PropagateAtLaunch: false
-        Value: "30Gi"
+        Value: "$data.NodePool.AvailableStorage 42949672960 0.95"
```

Comment:
nit: maybe add a comment what this number is? :)

Reply:
Good point, done.